onnxruntime/test/util/include/default_providers.h

```diff
@@ -33,6 +33,7 @@ std::unique_ptr<IExecutionProvider> DefaultCudaExecutionProvider();
 std::unique_ptr<IExecutionProvider> DefaultDnnlExecutionProvider(bool enable_arena = true);
 std::unique_ptr<IExecutionProvider> DefaultNupharExecutionProvider(bool allow_unaligned_buffers = true);
 std::unique_ptr<IExecutionProvider> DefaultTensorrtExecutionProvider();
```

Comment:
Could you please add a comment to "DefaultTensorrtExecutionProvider" and mark it as deprecated? I guess one is enough.

Reply:
I was wondering why can't we keep it? Right now several testcases are using this function. Also, if we later deprecate this function, we will need to write provider option initialization code which is lengthy each time under the situation we need default configuration.